### PR TITLE
Added fixes to post Route

### PR DESCRIPTION
**Pull request by robester0403**
What does this code do?
Fixes the frontend to work with the bugs from switching into AWS. 

What are potential issues?
Not sure how the rest of the code/branches is interacting with the backend.

### CODE REVIEW

File: src/utils/axios.js [hunk 1]
```diff
@@ -11,14 +11,15 @@ export const apiRequest = axios.create({
...
+    console.log(res);
     return res;
   } catch (err) {
     console.log("Error in sending heartbeat to server");
   }
 };
 
 export const getHeartBeat = async (userId, lookbackVal) => {
-  const lookback = lookbackVal ? lookbackVal : 1;
+  const lookback = lookbackVal ? "" : 1;
```
Comment (henryluan95) [inline src/utils/axios.js:22]: Could you explain what is lookback? and what does {params: {lookback: lookback} do for the API call? Thanks.

**Review by henryluan95 — APPROVED** [review]
Everything looks good to me. Just wondering about the lookback.

File: src/utils/axios.js (hunk 1 above)
Comment (robester0403) [reply src/utils/axios.js:22]: Lookback is the variable that allows you to get more than one previous data point, so lookback = 3 will give 3 data points. 

The above is actually legacy code back when lookback was different. 

Looked at the new server and made a new commit for how lookback should be working now in the current server environment.

**Review by FioraDeMarco — APPROVED** [review]
Was the bug coming from instances where there wasn't a look-back parameter?

**Pull request merged**